lib/Raven/Serializer.php

```diff
@@ -109,6 +109,12 @@ protected function serializeValue($value)
         if (is_null($value) || is_bool($value) || is_float($value) || is_integer($value)) {
             return $value;
         } elseif (is_object($value) || gettype($value) == 'object') {
+            // If the value is a callable and expects no parameters, we evaluate the
+            // callable value by calling it.
+            if (is_callable($value) && Raven_Util::getCallableParamNum($value) === 0) {
+                return $value();
```

Comment:
@ste93cry pardon my asking, but as a non-native English speaker I would like a clarification regarding

> you can't base your assumptions on the fact that any callable/method should be called

Do you mean that I should assume that all callables should not be evaluated unless there is heavy reason to do it (e.g. we should make the callable evaluation a more defined decision on the Sentry integrators part)?

Or something else?